docs/branches.md

```diff
@@ -11,23 +11,28 @@ pass a branch's name to commands that want a revision as argument. For example,
 `jj branch list` to list branches and `jj branch` to create, move, or delete
 branches. There is currently no concept of an active/current/checked-out branch.
 
-## Remotes
-
-Jujutsu records the last seen position on each remote (just like Git's
-remote-tracking branches). You can refer to these with
-`<branch name>@<remote name>`, such as `jj new main@origin`.
-
-A remote branch can be associated with a local branch of the same name. It's
-sometimes called a tracking branch. When you pull from a remote, any changes
-compared to the current record of the remote's state will be propagated to the
-tracking local branch. Let's say you run `jj git fetch --remote origin` and the
-remote's "main" branch has moved so its target is now ahead of the local record
-in `main@origin`. That will update `main@origin` to the new target. It will also
-apply the change to the local branch `main`. If the local target had also moved
-compared to `main@origin` (probably because you had run `jj branch set main`),
+## Remotes and tracked branches
+
+Jujutsu records the last seen position of a branch on each remote (just like
+Git's remote-tracking branches). This record is updated on every `jj git fetch`
+and `jj git push` of the branch. You can refer to the remembered remote branch
+positions with `<branch name>@<remote name>`, such as `jj new main@origin`. `jj`
+does not provide a way to manually edit these recorded positions.
+
+A remote branch can be associated with a local branch of the same name. This is
+called a **tracked branch**. When you pull a tracking branch from a remote, any
```

Comment:
fwiw, I'm not sure which one we should call a tracking or tracked.

Appears that Git defines these as follows:
* remote-tracking branch: remote branches stored in local repo
* tracking branch: local branch tracking a remote branch
* upstream branch: remote-tracking branch tracked by a local branch
https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches#_tracking_branches

Perhaps, we should avoid using "remote-tracking branch" because it's super confusing.

Reply:
I chose "tracked" in this pr to decrease confusion with remote-tracking.

Maybe we need a different word entirely. "Synced?" "Followed remote branch and following local branch"?

Reply:
For Git people: "downstreamed remote branch" or "mirrored downstream"

Reply:
> I chose "tracked" in this pr to decrease confusion with remote-tracking.

I think "tracked" (for the remote counterpart) is pretty okay. The command is `jj branch track <branch>@<remote>`, so the remote branch can be considered "tracked".

"Synced" sounds a bit odd to me because the branch can get out of sync. Btw, I use "synced" (internally in templater) to represent a branch that doesn't have `*` sigil.

Reply:
My one worry about "tracked" is that "local tracking branches" become inevitable and confusable with "remote-tracking branches". Perhaps it's ok if we always insert the word "local".

Reply:
Just commenting from the sidelines, I think "local tracking branches" is fine. Ideally we should have a term which doesn't overlap with Git.

Reply:
I added a commit with an explanation of all of these terms. We can still change the terminology if we come up with something.